autoModelCar:
  kspeed: 10
  kbreak: 20
  kidle: 2

  max_speed: 260
  max_angle: 80 # + 90
  reverse_offset: -200 
  reverse_speed: -120

